public/app/features/users/TokenRevokedModal.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Button, InfoBox, Portal, stylesFactory, useTheme } from '@grafana/ui';
+import { getModalStyles } from '@grafana/ui/src/components/Modal/getModalStyles';
+import { css, cx } from 'emotion';
+
+const getStyles = stylesFactory(() => {
+  return {
+    infobox: css`
+      margin-bottom: 0;
+    `,
+    text: css`
+      margin: 8px 0 16px;
+    `,
+    backdrop: css`
+      background-color: #0b0d10;
+      opacity: 0.8;
+    `,
+  };
+});
+
+interface Props {
+  maxConcurrentSessions?: number;
+}
+
+export const TokenRevokedModal = (props: Props) => {
+  const theme = useTheme();
+
+  const styles = getStyles();
+  const modalStyles = getModalStyles(theme);
+
+  const showMaxConcurrentSessions = props.maxConcurrentSessions && props.maxConcurrentSessions > 0;
+
+  const redirectToLogin = () => {
+    window.location.reload();
+  };
+
+  return (
+    <Portal>
+      <div className={modalStyles.modal}>
+        <InfoBox title="You have been automatically logged out" severity="warning" className={styles.infobox}>
+          <div className={styles.text}>
+            <p>
+              Your session token was automatically revoked because you have reached
+              <strong>
+                {' '}
+                the maximum number of
+                {showMaxConcurrentSessions && <span> {props.maxConcurrentSessions} </span>}
+                concurrent sessions{' '}
+              </strong>
```

Comment:
The `concurrent sessions` part of the text is missing a space before. Maybe change this to:
```suggestion
            <strong>
                {` the maximum number of ${
                  showMaxConcurrentSessions ? props.maxConcurrentSessions : ''
                } concurrent sessions `}
              </strong>
```